docker/build-tools/Dockerfile

```diff
@@ -359,9 +359,10 @@ ENV GO111MODULE=on
 ENV GOPROXY=https://proxy.golang.org
 ENV GOSUMDB=sum.golang.org
 ENV GOROOT=/usr/local/go
-ENV GOPATH=/home/go
-ENV GOCACHE=/home/gocache
-ENV PATH=/usr/local/go/bin:/home/go/bin:/usr/local/google-cloud-sdk/bin:$PATH
+ENV GOPATH=/go/base
```

Comment:
this may be a bit confusing, as gopath normal is /go or ~/go. Can we make this just /go and things will work?

Reply:
Yeah, we can do that.